### PR TITLE
Api mgmt

### PR DESCRIPTION
**Pull request by ReenuSaluja**

### CODE REVIEW

**Comment by jmspring** [comment]
@ReenuSaluja - there are merge issues in the pull request.  Can you resolve those?  That was just a first pass.  I will review fully a bit later.

File: cluster/environments/azure-multiple-clusters-waf-tm-apimgt/aks-centralus-variables.tf [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+variable "central_resource_group_location" {
+  type = "string"
+}
+
+variable "gitops_central_path" {
+  type = "string"
+}
+<<<<<<< HEAD
```
Comment (timfpark) [inline cluster/environments/azure-multiple-clusters-waf-tm-apimgt/aks-centralus-variables.tf]: Fix merge conflicts

File: cluster/environments/azure-multiple-clusters-waf-tm-apimgt/aks-centralus.tf [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  address_space           = "${var.central_address_space}"
+  subnet_prefixes         = "${var.central_subnet_prefixes}"
+  tags = {
+    environment = "azure-multiple-clusters"
+  }
+}
+
+# Creates aks cluster
+module "central_aks" {
+  source = "../../azure/aks"
```
Comment (timfpark) [inline cluster/environments/azure-multiple-clusters-waf-tm-apimgt/aks-centralus.tf]: Migrate this portion to use the aks-gitops module over a remote git source (see azure-single-keyvault for example).

File: cluster/environments/azure-multiple-clusters-waf-tm-apimgt/aks-eastus-variables.tf [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+variable "east_resource_group_location" {
+  type    = "string"
+}
+
+variable "gitops_east_path" {
+  type = "string"
+}
+<<<<<<< HEAD
```
Comment (timfpark) [inline cluster/environments/azure-multiple-clusters-waf-tm-apimgt/aks-eastus-variables.tf]: Fix merge conflict.

File: cluster/environments/azure-multiple-clusters-waf-tm-apimgt/aks-centralus.tf (hunk 2 above)
Comment (ReenuSaluja) [reply cluster/environments/azure-multiple-clusters-waf-tm-apimgt/aks-centralus.tf]: This terraform template is from the existing api-mgmt branch. I will check with Nate on what needs to be done here.

File: cluster/environments/azure-multiple-clusters-waf-tm-apimgt/aks-centralus-variables.tf (hunk 1 above)
Comment (ReenuSaluja) [reply cluster/environments/azure-multiple-clusters-waf-tm-apimgt/aks-centralus-variables.tf]: Resloved the conflict

File: cluster/environments/azure-multiple-clusters-waf-tm-apimgt/aks-eastus-variables.tf (hunk 3 above)
Comment (ReenuSaluja) [reply cluster/environments/azure-multiple-clusters-waf-tm-apimgt/aks-eastus-variables.tf]: Fixed merge conflict

Comment (ReenuSaluja) [reply cluster/environments/azure-multiple-clusters-waf-tm-apimgt/aks-eastus-variables.tf]: Still see some conflicts. Will resolve it tomorrow

**Comment by ReenuSaluja** [comment]
we are working on the recommendations . Closing this pull request for now. I will raise a new pull request once done with the changes.

**Pull request closed without merging**